be/src/common/config.h

```diff
@@ -892,6 +892,9 @@ CONF_mInt16(pipeline_short_query_timeout_s, "20");
 // Will remove after fully test.
 CONF_Bool(enable_index_apply_preds_except_leafnode_of_andnode, "true");
 
+// buffered prefetch
+CONF_mInt32(prefetch_single_buffer_size_mb, "4");
```

Comment:
I think we can remove this config, use a fixed value, if there is no need to modify this.